components/list/list-item.js

```diff
@@ -27,7 +27,6 @@ class ListItem extends RtlMixin(LitElement) {
 			breakpoints: { type: Array },
 			disabled: {type: Boolean },
 			href: { type: String },
-			illustrationOutside: { type: Boolean, attribute: 'illustration-outside' },
```

Comment:
There's a bunch of CSS selectors that hook into this attribute.  ex.  `:host([illustration-outside]) ...` that should also be removed.

Reply:
Ah, you bet me to it. See my comment for the CSS to be removed.